docs/key-issues/multiple_architectures.md

```diff
@@ -0,0 +1,266 @@
+# Platforms with multiple CPU architectures
+
+One important subset of ABI concerns is the CPU architecture for which a binary
+artefact has been built. Attempting to run a binary on hardware that doesn't
+match the CPU architecture (or architecture variant[^1]) for which the binary
+was built will generally lead to crashes, even if the ABI being used is
+otherwise compatible.
+
+[^1]:
+    E.g., the x86-64 architecture has a range of well-known extensions, such as
+    SSE, SSE2, SSE3, AVX, AVX2, AVX512, etc.
+
+## Current state
+
+Historically, it could be assumed that an executable or library would be
+compiled for a single CPU architecture. On the rare occasion that an operating
+system was available for mulitple CPU architectures, it became the
```

Comment:
This sentence still needs a rewrite, an OS being supported on multiple CPU architectures is the rule rather than the exception.

Reply:
I was referring more to the macOS situation of user-space actively expecting multiple architectures, rather than the more general "Linux is available on almost everything"; however, I agree this is ambiguous. I'll update the text.